Out-of-source build from git code is broken

```
git reset --hard origin/main 
cmake -B builddir
cmake --build builddir --target manpage
```

leads to

```
[100%] Building lfortran manpage
pandoc: lfortran.md: withBinaryFile: does not exist (No such file or directory)
gmake[3]: *** [doc/man/CMakeFiles/manpage.dir/build.make:73: doc/man/lfortran.1] Error 1
```

I introduced this bug in #2934.

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -293,6 +293,48 @@ jobs:
         env:
           SSH_PRIVATE_KEY_WASM_BUILDS: ${{ secrets.SSH_PRIVATE_KEY_WASM_BUILDS }}
 
+  debug_outOfSource:
+    name: Check Out-of-Source Debug build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: ci/environment_linux.yml
+          create-args: >-
+            python=3.10
+
+      - uses: hendrikmuhs/ccache-action@main
+        with:
+          variant: sccache
+          key: ${{ github.job }}-${{ matrix.os }}
+
+      - name: Build Linux
+        shell: bash -e -l {0}
+        run: |
+            ./build0.sh
+            export CXXFLAGS="-Werror"
+            cmake -S . -B builddir -GNinja \
+              -DCMAKE_BUILD_TYPE=Debug \
+              -DWITH_LLVM=yes \
+              -DLFORTRAN_BUILD_ALL=yes \
+              -DWITH_STACKTRACE=no \
+              -DWITH_RUNTIME_STACKTRACE=yes \
+              -DCMAKE_PREFIX_PATH="$CONDA_PREFIX" \
+              -DCMAKE_INSTALL_PREFIX=`pwd`/inst \
+              -DCMAKE_C_COMPILER_LAUNCHER=sccache \
+              -DCMAKE_CXX_COMPILER_LAUNCHER=sccache
+
+            cmake --build builddir  -j16 --target install
+
+      - name: Test Linux
+        shell: bash -e -l {0}
+        run: ctest --output-on-failure
+        working-directory: builddir
+
   debug:
     name: Check Debug build
     runs-on: ubuntu-latest

--- a/doc/man/CMakeLists.txt
+++ b/doc/man/CMakeLists.txt
@@ -8,7 +8,7 @@ elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/lfortran.md)
   find_package_handle_standard_args(PANDOC REQUIRED_VARS PANDOC_EXECUTABLE)
   if(PANDOC_FOUND)
     add_custom_command(OUTPUT lfortran.1
-      COMMAND ${PANDOC_EXECUTABLE} --standalone --to man lfortran.md -o lfortran.1
+      COMMAND ${PANDOC_EXECUTABLE} --standalone --to man ${CMAKE_CURRENT_SOURCE_DIR}/lfortran.md -o lfortran.1
       COMMENT "Building lfortran manpage"
       DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/lfortran.md)
     add_custom_target(manpage ALL DEPENDS lfortran.1)

